### PR TITLE
[author] [new] add fomantic-ui w/ git auto-update via single package.json

### PR DESCRIPTION
**Pull request by hammy2899**
Pull request for issue: N/A
Related issue(s): N/A

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/fomantic/Fomantic-UI
* Official website (optional, not the repository): https://fomantic-ui.com/
* NPM package url (optional): https://www.npmjs.com/package/fomantic-ui
* License and its reference: [MIT](https://github.com/fomantic/Fomantic-UI/blob/master/LICENSE.md)
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 11
  * Count of stars: 60
  * Count of forks: 6
* NPM download stats (optional):
  * Downloads in the last day: 108
  * Downloads in the last week: 352
  * Downloads in the last month: 964

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/fomantic-ui/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "description": "Community fork of the popular Semantic-UI framework",
+  "keywords": [
+    "semantic",
+    "fomantic",
+    "framework",
+    "ui"
+  ],
+  "author": "hammy2899",
+  "license": "MIT",
+  "filename": "semantic.min.js",
```
Comment (sashberd) [inline ajax/libs/fomantic-ui/package.json]: This field should come after name property

File: ajax/libs/fomantic-ui/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "fomantic-ui",
+  "description": "Community fork of the popular Semantic-UI framework",
+  "keywords": [
+    "semantic",
+    "fomantic",
+    "framework",
+    "ui"
+  ],
+  "author": "hammy2899",
```
Comment (sashberd) [inline ajax/libs/fomantic-ui/package.json]: Lets add homepage property as: https://fomantic-ui.com

**Comment by hammy2899** [comment]
@sashberd I only want to host the GitHub release since NPM has a few versions (as mentioned) which weren't releases as per say the ones on GitHub.

**Comment by ColinFrick** [comment]
Is anything else needed for this to move forward?

**Comment by xFirez** [comment]
Any news?

**Comment by hammy2899** [comment]
@sashberd Do you have any idea when this will be merged?

**Comment by sashberd** [comment]
@hammy2899 I have no idea. Only @extend1994 @sufuf3 @PeterDaveHello have permissions to merge PR in cdnjs. Thus, you should wait for them. 
Moreover, the cdnjs maintenance activity is down almost to  0, so do not expect to merge this PR soon 😞

**Comment by hammy2899** [comment]
@sashberd Ok thanks for the info 👍

File: ajax/libs/fomantic-ui/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  "name": "fomantic-ui",
+  "filename": "semantic.min.js",
+  "description": "Community fork of the popular Semantic-UI framework",
+  "keywords": [
+    "semantic",
+    "fomantic",
+    "framework",
+    "ui"
+  ],
+  "author": "hammy2899",
```
Comment (extend1994) [inline ajax/libs/fomantic-ui/package.json]: Maybe also Jack Lukic? Refer https://github.com/fomantic/Fomantic-UI/blob/master/package.json#L7

File: ajax/libs/fomantic-ui/package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  "homepage": "https://fomantic-ui.com",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fomantic/Fomantic-UI.git"
+  },
+  "autoupdate": {
```
Comment (extend1994) [inline ajax/libs/fomantic-ui/package.json]: Would you like to use npm auto-update instead? I saw there are v2.3.1/2.3.2 on it but not on GitHub.

Comment (hammy2899) [reply ajax/libs/fomantic-ui/package.json]: Already discussed https://github.com/cdnjs/cdnjs/pull/12942#issuecomment-418687983

File: ajax/libs/fomantic-ui/package.json (hunk 3 above)
Comment (hammy2899) [reply ajax/libs/fomantic-ui/package.json]: Jack has no involvement with FUI.

File: ajax/libs/fomantic-ui/package.json (hunk 4 above)
Comment (extend1994) [reply ajax/libs/fomantic-ui/package.json]: Sorry for the missing

File: ajax/libs/fomantic-ui/package.json (hunk 3 above)
Comment (extend1994) [reply ajax/libs/fomantic-ui/package.json]: He had? https://github.com/fomantic/Fomantic-UI/graphs/contributors

Comment (hammy2899) [reply ajax/libs/fomantic-ui/package.json]: @extend1994 Any news?

Comment (extend1994) [reply ajax/libs/fomantic-ui/package.json]: Sorry for the late reply, missed this one. 
Didn't know the situation you described. Both are fine to me :)

**Pull request merged**